CONTRIBUTORS.txt

```diff
@@ -269,3 +269,5 @@ contributors:
 * Svetoslav Neykov: contributor
 
 * Federico Bond: contributor
+
+* Thomas Hisch: contributor
```

Comment:
If you patch is based on the one in #1189, please acknowledge that here too :)

Reply:
Sure. Thx for pointing that out.